### DISCUSSION TITLE
Hosted Server connection

### DISCUSSION THREAD

**Discussion by monejoel** [BearWare/TeamTalk5#1480] [General]
How do I get my TeamTalk service hosted on my shared hosting?
Please I need a guide.

**Comment by ChrisDuffley** [comment]
Actually, instead of doing it all manually, perhaps you should try [this guide from Doug Lee](https://dlee.org/teamtalk/tt) which explains on the easiest way in the world to do it on Linux. Thanks to @dglee42 for his excellent work!

**Reply by tech10** [reply]
Your indicated guide is a good one if you aren't on shared hosting, but able to manage your own system.

Typically, in a shared hosting environment, a comuter is running a number of different websites. If a person has any shell access to their account, they likely won't be able to run their own TeamTalk server.

@monejoel I would judge the ability to get TeamTalk servers running on a shared hosting environment to be highly unlikely, you'd be better off getting a VPS (Virtual Private Server). If you're not able to do that, there are plenty of online resources that can provide TeamTalk server hosting, especially if you have no idea how to manage a VPS, what it is, how to administrate Linux or which ever operating system you choose to use, etc.
